sympy/functions/special/error_functions.py

```diff
@@ -1655,8 +1655,14 @@ class Si(TrigonometricIntegral):
 
     _trigfunc = sin
     _atzero = S(0)
-    _atinf = pi*S.Half
-    _atneginf = -pi*S.Half
+
+    @classmethod
+    def _atinf(cls):
+        return pi*S.Half
+
```

Comment:
S.Half is a call into SymPy, which would not be "initialized enough" to evaluate properly.
Changing these attributes into class methods would make sure that no SymPy code would be called before SymPy's modules would be fully installed.
(_atzero happens to work. Maybe it should be turned into a function just for consistency with that "don't call into SymPy in the class body" rule, if we adopt that.)